### DISCUSSION TITLE
Contributor agreement issue

### DISCUSSION THREAD

**Discussion by jamesblacklock** [mempool/mempool#4524] [Q&A] [answered]
I created this PR: https://github.com/mempool/mempool/pull/4523

I believe I followed the instructions for the contributor agreement, but the bot is saying I have no agreement on file.

Anyone know what I'm doing wrong?

**Comment by softsimon** [comment, marked as answer]
I think the bug is that it can't detect the agreement if it's in the same PR, so just ignore the warning that's wrong.

@knorrium perhaps knows more.

**Reply by rawgir-7Xoqca-nezgej** [reply]
2176
